src/Sentry/ExceptionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using Sentry.Internal;
+
+namespace Sentry
```

Comment:
I think an extension method to `Exception` coming from your Crash reporting tool will not surprising and justifies adding to the global namespace. We need to continue to take special care with the name to avoid any potential conflict or ambiguity as you did in this PR (Having Sentry in the method name will be a must).

 ```suggestion
```